src/MassTransit.RabbitMqTransport/IExchangeTypeDeterminer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MassTransit.RabbitMqTransport
+{
+    public interface IExchangeTypeDeterminer
```

Comment:
I was struggling to find a suitable name for this interface.

In my view a name should indicate the type of behaviour that can be found in the interface (or class)
In this case the interface facilitates getting the correct type of an exchange (fanout, topic, direct, custom exchange types) from a (exchange)name that is passed to it.

The best name I could think of for that behaviour was determining the exchange type.
Which I translated to IExchangeTypeDeterminer.

Currently I am still unable to think of a better name for the interface.

Perhaps, after reading the story above, you have a suggestion for a better name?